aten/src/ATen/native/native_functions.yaml

```diff
@@ -8382,6 +8382,17 @@
 - func: special_erfinv.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   python_module: special
 
+- func: special_i0e(Tensor self) -> Tensor
+  python_module: special
+  variants: function
+  dispatch:
+    CPU, CUDA: special_i0e
+
+- func: special_i0e.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  python_module: special
+  dispatch:
+    CPU, CUDA: special_i0e_out
```

Comment:
Please consider writing this as a structured kernel! https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md

Reply:
Done